test/integration/dashboard.spec.js

```diff
@@ -244,4 +244,32 @@ describe('Dashboard', () => {
         .then(() => expect(dashboardPage.timelineStatusSize('result')).to.equal(0))
     );
   });
+  describe('focus on job', () => {
+    let savedJob;
+
+    const createJob = (attributes) =>
+      new JobsModel(Object.assign({}, jobData, attributes)).save();
+
+    describe('status', () => {
+      before(function () {
+        return helper.cleanDb()
+          .then(() => createJob())
+          .then((job) => { savedJob = job; });
+      });
+
+      it('should display details of the job in focus', () =>
```

Comment:
wouldn't be better to change this scenario to `should display details of the job after creation`